rocky/rocky/auth/remote_user.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+from django.conf import settings
+from django.contrib.auth.backends import RemoteUserBackend as BaseRemoteUserBackend
+from django.contrib.auth.models import Group
+from tools.models import Organization, OrganizationMember
+
+logger = logging.getLogger(__name__)
+
+
+class RemoteUserBackend(BaseRemoteUserBackend):
+    """
+    Custom RemoteUserBackend that adds users to default organizations and groups.
+    """
+
+    def configure_user(self, request, user, created=True):
+        if not user.is_superuser and settings.REMOTE_USER_DEFAULT_ORGANIZATIONS:
+            user_orgs = [o.name for o in user.organizations]
+            for item in settings.REMOTE_USER_DEFAULT_ORGANIZATIONS:
+                organization_name, group_name = item.split(":")
```

Comment:
There are no real safeguards here now; perhaps we want to add a little check on the setting format somewhere to avoid a small configuration mistake causing no one to be able to log in anymore?

Reply:
I've added exception handling so errors will be logged without breaking the login flow, do you think that's sufficient?